wp-content/themes/pawar2018/js/scripts/app.js

```diff
@@ -35,5 +35,12 @@
     })
   }
 
+  window.onload = function init() {
+    var vidDefer = document.getElementById('featured-video');
+    if(vidDefer.hasAttribute('data-src')) {
```

Comment:
Should check for the existence of `vidDefer` here as well, as it only exists on the home page (which currently throws an error on other pages). Something like:

```
if (vidDefer && vidDefer.hasAttribute('data-src')) {
```